movies-app/src/components/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import './MovieForm.css';
+
+const MovieForm = ({ movieInfo }) => {
+    const [formData, setFormData] = useState(movieInfo || {});
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData((prevData) => ({
+            ...prevData,
+            [name]: value,
+        }));
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log(formData);
+        // onSubmit(formData);
+        // onClose();
+    };
+
+    const resetForm = () => {
+        setFormData(movieInfo || {})
+    }
+
+    return (
+            <form onSubmit={handleSubmit}>
+                <div className="row">
+                    <div className="col-8">
+                        <div className="mb-4">
+                            <label for="title" className="form-label">Title</label>
+                            <input type="text" className="form-control" id="title" name="title" onChange={handleChange} value={formData.title} placeholder="Title" />
+                        </div>
+                    </div>
+
+                    <div className="col-4">
+                        <div className="mb-4 position-relative">
+                            <label for="release_date" className="form-label">Release Date</label>
+                            <input type="text" className="form-control icon-input" id="release_date"   onChange={handleChange} name="release_date" value={formData.release_date} placeholder="Select Date" />
+                            <div className="icon">
```

Comment:
SVG's can be put in a separate folder so that they can be reused

Reply:
Moved to assets folder